t/lib/server.lua

```diff
@@ -115,6 +115,23 @@ function _M.mock_zipkin()
         if not span.traceId then
             ngx.exit(400)
         end
+
+        if not span.localEndpoint then
+            ngx.exit(403)
+        end
+
+        if span.localEndpoint.serviceName ~= 'APISIX' and span.localEndpoint.serviceName ~= 'apisix' then
```

Comment:
Why judge both case and capital?

Reply:
because, by default it is `APISIX`, ans my test case want to test some no-default value.